src/elements/featured-videos.html

```diff
@@ -0,0 +1,330 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/plastic-image/plastic-image.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../bower_components/polymer/lib/mixins/gesture-event-listeners.html">
+<link rel="import" href="../redux-mixin.html">
+<link rel="import" href="flex-layout-attr.html">
+<link rel="import" href="video-dialog.html">
+
+<dom-module id="featured-videos">
+  <template>
+    <style is="custom-style" include="shared-styles flex flex-alignment"></style>
+
+    <style>
+       :host {
+        display: block;
+      }
+
+      .videos-wrapper {
+        padding: 0;
+      }
+
+      .header {
+        margin: 24px auto;
+        padding: 0 16px;
+      }
+
+      .title {
+        font-size: 24px;
+      }
+
+      .icon-label {
+        margin-left: 10px;
+      }
+
+      .video-icon {
+        margin-left: 0;
+        padding-left: 0;
+      }
+
+      .video-list {
+        position: relative;
+        margin: 0 auto;
+        max-width: 1280px;
+        height: 250px;
+        overflow: hidden;
+        margin-left: -15px;
+      }
+
+      .videos {
+        transition: all 0.4s cubic-bezier(0, 0, 0.2, 1);
```

Comment:
let's define and use one animation everywhere. Right now, we have defined animation in `shared-styles` https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/src/elements/shared-styles.html#L27

We can discuss which animation will work for us in the best way